kind_with_registry.sh

```diff
@@ -15,6 +18,7 @@ reg_port='5001'
 if [ "$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || true)" != 'true' ]; then
   docker run \
     -d --restart=always -p "127.0.0.1:${reg_port}:5000" -e REGISTRY_STORAGE_DELETE_ENABLED=true --name "${reg_name}" \
+     --network bridge \
```

Comment:
would it still be needed if we use podman here instead of docker?

Reply:
> would it still be needed if we use podman here instead of docker?

yes , we would still need to add this parameter , 
in my setup i'm using podman-docker which is podman behind the scene.
```
cat /usr/bin/docker
#!/usr/bin/sh
[ -f /etc/containers/nodocker ] || \
echo "Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg." >&2
exec /usr/bin/podman "$@"
```